explore/src/main/scala/explore/observationtree/ObsSummaryTable.scala

```diff
@@ -254,9 +275,14 @@ object ObsSummaryTable:
           obsColumn(
             DurationColumnId,
             _.execution.map(_.programTimeEstimate)
-          ).setCell: cell =>
+          ).setCell { cell =>
             cell.value.map:
               _.orSpinner(_.map(_.toHoursMinutes).orEmpty)
+          }.setSortingFn { (a, b, c) =>
+            val aʹ: Option[Pot[Option[TimeSpan]]] = a.getValue(c)
+            val bʹ: Option[Pot[Option[TimeSpan]]] = b.getValue(c)
+            tsOrder.compare(aʹ.flatMap(_.toOption).flatten, bʹ.flatMap(_.toOption).flatten)
```

Comment:
Does it work if we just `aʹ.flatMap(_.toOption).flatten.compareTo(bʹ.flatMap(_.toOption).flatten)` and avoid defining `tsOrder`?

Also, consider an extension for the `.flatMap(_.toOption).flatten` ?